tests/templates/script/fixtures/pagination.hbs.js

```diff
@@ -0,0 +1,13 @@
+ANSWERS.addComponent("Pagination", Object.assign({}, {
+  container: "#js-answersPagination",
+  onPaginate: (newPageNumber, oldPageNumber, totalPages) => {
+    if (window.parentIFrame) {
+      const paginateMessage = { action: 'paginate' };
+      window.parentIFrame.sendMessage(JSON.stringify(paginateMessage));
+    } else {
+      document.documentElement.scrollTop = 0;
+      document.body.scrollTop = 0; // Safari
+    }
+  },
+    verticalKey: "testKey",
+}, {}));
```

Comment:
There's no data here but that's probably fine. We only really need data for the partials using the new helper